templates/deployment.yaml

```diff
@@ -163,6 +167,11 @@ spec:
         emptyDir: {}
       - name: app-config
         emptyDir: {}
+      - name: enginelog  
```

Comment:
> I don't understand the check for GCP, it should have the same path. What I was referring to was a conditional for logstreaming enabled. If it's enabled, use hostpath, otherwise use emptyDir.

GCP host path is different from AWS and AZURE host path so i added this condition .

Reply:
Then let's expose the path as a configurable parameter so we aren't restricted to only those two paths
```
- name: enginelog 
  {{- if .Values.logstream.enabled }} 
  hostPath:
    path: {{ .Values.logstream.hostPath }}
    type: '' 
  {{- else }}
  emptyDir: {}
  {{- end }}
```

Reply:
updated the value file to hold default host path